demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -226,7 +226,7 @@ <h1 class="h3 mb-3 font-weight-normal text-center">Select devices</h1>
   </div>
   <audio id="meeting-audio" style="display:none"></audio>
   <div id="meeting-container" class="container-fluid h-100" style="display:flex; flex-flow:column">
-    <div class="row mb-3 mb-lg-0" style="flex-basis: auto;">
+    <div class="row mb-3 mb-lg-0" style="flex: unset;">
```

Comment:
Why are we changing the styling in meeting V2 ?

Reply:
Is it to be consistent with the meeting V1 ?

Reply:
yeah, flex sets three different properties: flex-grow, flex-shrink, and flex-basis. 

Initially I thought we could just reset flex-basis, but I think it would be better to reset all the flex properties to default by doing unset. (this is how it was prior to the 4.5.1 bootstrap change)